instances/latest/commonCoordinateSpaces/MarmosetNMT.jsonld

```diff
@@ -0,0 +1,25 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/commonCoordinateSpace/MarmosetNMT",
+    "@type": "https://openminds.om-i.org/types/CommonCoordinateSpace",
+    "abbreviation": "MarmosetNMT",
+    "author": null,
+    "custodian": null,
+    "description": "The Nencki\u2013Monash (NM) template v1.0 (2020) represents a computational morphological average of 20 gender-balanced young adult brains of the Common Marmoset monkey (Callithrix jacchus), derived from 3D reconstructions based on Nissl-stained serial sections. The dataset includes a Nissl-like color anatomical reference rendered at 50 \u00b5m isotropic resolution, estimates of cortical thickness, and spatial transformations between the NM template and selected other templates of the marmoset brain to enhance interoperability. The template is embedded in stereotaxic coordinates defined following Paxinos et al. (2012) definition.",
```

Comment:
Since you are referring here to the exact version this is not a description of a version independent summary. Suggestion keep this one highlevel and short. Use the description you have for the MarmosetNMT_v1

```suggestion
    "description": "The Nencki\u2013Monash (NM) template represents a computational morphological average of multiple gender-balanced young adult brains of the Common Marmoset monkey (Callithrix jacchus), derived from 3D reconstructions based on Nissl-stained serial sections.",
```